halyard-web/src/main/java/com/netflix/spinnaker/halyard/controllers/v1/PersistentStorageController.java

```diff
@@ -82,6 +83,49 @@
     builder.setRevert(() -> halconfigParser.undoChanges());
     builder.setSave(() -> halconfigParser.saveConfig());
 
-    return DaemonTaskHandler.submitTask(builder::build, "Edit storage settings");
+    return DaemonTaskHandler.submitTask(builder::build, "Set current persistent store");
+  }
+
+  @RequestMapping(value = "/persistentStore/{persistentStoreName:.+}", method = RequestMethod.GET)
+  DaemonTask<Halconfig, PersistentStore> getPersistentStore(@PathVariable String deploymentName,
```

Comment:
How do you pick a persistent store to use? Do you have enable & disable explicitly?